packager/mpd/base/xml/xml_node.h

```diff
@@ -205,8 +205,12 @@ class RepresentationXmlNode : public RepresentationBaseXmlNode {
   /// Adds fields that are specific to VOD. This ignores @a media_info fields
   /// for Live.
   /// @param media_info is a MediaInfo with VOD information.
+  /// @param use_segment_list is a param that instructs the xml writer to use SegmentList instead of SegmentBase.
+  /// @param target_segment_duration is a param that provides the user give value for --segment_duration.
```

Comment:
Can you keep the number of characters per line to 80? 

For `target_segment_duartion`, please also mention that it is only used when `use_segment_list` is true.